`QuicConfiguration.secrets_log_file` has an invalid default value

See:

https://github.com/aiortc/aioquic/blob/7f27a0b8fd2c4ab755c5a7ba32b4d432dda305dd/src/aioquic/quic/configuration.py#L73

This means if you try and initialise `QuicConfiguration` yourself with `secrets_log_file=None` you'll get an error like:

```
tools/webtransport/h3/webtransport_h3_server.py:547: error: Argument "secrets_log_file" to "QuicConfiguration" has incompatible type "TextIOWrapper[_WrappedBuffer] | None"; expected "TextIO"  [arg-type]
```

The only workarounds are to not specify it (and rely on the default) or to add an `#  type: ignore[arg-type]`.

--- a/src/aioquic/quic/configuration.py
+++ b/src/aioquic/quic/configuration.py
@@ -70,7 +70,7 @@ class QuicConfiguration:
     The :class:`~aioquic.quic.logger.QuicLogger` instance to log events to.
     """
 
-    secrets_log_file: TextIO = None
+    secrets_log_file: Optional[TextIO] = None
     """
     A file-like object in which to log traffic secrets.
 

